flink-core/src/main/java/org/apache/flink/configuration/HistoryServerOptions.java

```diff
@@ -89,6 +89,16 @@
 			.withDescription("Enable HTTPs access to the HistoryServer web frontend. This is applicable only when the" +
 				" global SSL flag security.ssl.enabled is set to true.");
 
+	/**
+	 * Specifies whether the HistoryServer should process job archive deletion. If this option is
+	 * enabled then deleted job archives are also deleted from HistoryServer.
+	 */
+	public static final ConfigOption<Boolean> HISTORY_SERVER_CLEANUP_EXPIRED_JOBS =
+		key("historyserver.web.clean-expired-jobs")
```

Comment:
the web prefix doesn't make sense here; web options are exclusively for the WebUI portion of the HS. `historyserver.archive` is a more appropriate config key.

Reply:
Prefix changed and constant moved next to other `historyserver.archive` config options. Thnak you